lib/tasks/one_off/add_missing_contracts_2896.rake

```diff
@@ -0,0 +1,62 @@
+module OneOff
+  MISSING_CONTRACTS_DATES_2896 = [
+     [2025, 1],
+     [2025, 2],
+     [2025, 3],
+     [2025, 4],
+    [2025, 5],
+    [2025, 6], # July 2025 is already created
+    [2025, 8],
```

Comment:
```suggestion
    [2025, 5],
    [2025, 6],
    # July 2025 is already created
    [2025, 8],
```

Reply:
makes it clearer I think - that, or make the comment consistently on the one after the missing month.